google-cloud-spanner/src/main/java/com/google/cloud/spanner/Database.java

```diff
@@ -88,6 +88,20 @@ public OperationFuture<Void, UpdateDatabaseDdlMetadata> updateDdl(
     return dbClient.updateDatabaseDdl(instance(), database(), statements, operationId);
   }
 
+  /**
+   * Updates a database in a Cloud Spanner instance
```

Comment:
nit:
```suggestion
   * Updates the schema of a database in a Cloud Spanner instance
```

Reply:
Just to be sure: This method only updates the schema of the database, right? Or does it do anything additional?

Reply:
Updated the comment as per the suggestion.
Yes @olavloite. This method updates the schema of the database which also includes updating schema of the Proto Bundles which lie in that database.